File: PROGRAMMERS/LEVEL1/이상한 문자 만들기.py
def solution(s):
    strList = list(map(str, s.split(' ')))
    
    answer = ''
    
    for st in strList:
        for i in range(len(st)):
            if i % 2 == 0: answer += st[i].upper()
            else: answer += st[i].lower()
        answer += ' '
        
    return answer[:-1]

File: PROGRAMMERS/LEVEL1/이상한 문자 만들기.java
// 공백 공백 공백!!! *split(" ", -1), trim(X)*
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer[] = s.split(" ", -1);
        String strAnswer = "";
        
        for (int i = 0;i < answer.length; i++) {
            String str = answer[i];
            for (int j = 0; j < str.length(); j++) {
                if (j % 2 == 0) {
                    strAnswer += Character.toUpperCase(str.charAt(j));
                }
                else {
                    strAnswer += Character.toLowerCase(str.charAt(j));
                }
            }
            if (i == answer.length - 1) {break;}
            strAnswer += " ";
        }
        return strAnswer;
    }
}
